app/assets/stylesheets/atoms/_variables.scss

```diff
@@ -14,6 +14,7 @@ $color-lavender: #C2C2FF;
 $color-purple: #3d3de1;
 $color-default-purple: #4c2c92;
 $color-venice-blue: #03578c;
+$color-stimulus-blue: #89DDEF;
```

Comment:
Non-blocking, but in general it's good to make sure we have a vocabulary with design about colors. Is this what Anu would call it?

Reply:
Anu called it "stimulus blue" :)